### ISSUE TITLE
is there the inference.py？

### ISSUE THREAD

**Issue by huacilang** [ID56/Torch-KWT#3]

**Comment by ID56** [comment]
Hi @huacilang. No currently there is no separate inference.py. However, you can take a look at the [colab tutorial]( https://colab.research.google.com/github/ID56/Torch-KWT/blob/main/notebooks/Torch_KWT_Tutorial.ipynb) -- particularly, the [Using Pre-trained Models for Inference](https://colab.research.google.com/github/ID56/Torch-KWT/blob/main/notebooks/Torch_KWT_Tutorial.ipynb#scrollTo=-fSPsrlIUAEz) section, which shows how to use a pretrained model to get predictions for random clips in the test set.

I will also open a PR soon, which will add two files:
- `inference.py` - Will run inference on short ~1s audio clips like the ones in Speech Commands Dataset
- `inference_window.py` - Will run inference on longer audio clips (e.g. 10s) by moving a "sliding window" across the audio array.

**Comment by ID56** [comment]
Hi, sorry it took a while. While the inference-script branch was ready a long time back, didn't have time to update the tutorial notebook and merge the PR in the past couple of weeks!

The main branch now has `inference.py` and `window_inference.py`.

You can see an actual example of using them in the [Using Pre-trained Models for Inference](https://colab.research.google.com/github/ID56/Torch-KWT/blob/main/notebooks/Torch_KWT_Tutorial.ipynb#scrollTo=-fSPsrlIUAEz) section of the updated colab tutorial.

<details>
<summary>snapshot</summary>

![image](https://user-images.githubusercontent.com/38450043/131225293-45f241d0-a22d-49e7-9c67-4b15b2aa1ed7.png)

</details>

**Comment by huacilang** [comment]
ok ，I see , already very soon.
thanks a lot
